### ISSUE TITLE
creature and push items

### ISSUE THREAD

**Issue by renanluis90** [TwistedScorpio/Nostalrius#40]
I remeber when i lose some items inside a parcel, because a demon skeleton push my parcel like 3x and the parcel

**Comment by emil92b** [comment]
not sure what you talking about, but regarding pushing/destroying objects i can confirm tfs has it wrong.

**Comment by renanluis90** [comment]
Yes, in my opinion, every time the object is pushed to another tile, the count is renewed, because of that the function is not working.

**Comment by renanluis90** [comment]
I tested on Realots
https://streamable.com/hjzxg
0:41 seconds u can see, the bandit deleted the box, but with BLOCKHIT effect ^^ is the correct.

**Comment by emil92b** [comment]
here is some old gifs i had (tests from experimental), mby you can get some information from them too

1 crate + player block - https://gyazo.com/4b8ebad8b7d7c071c549983a250d7c25
1 crate + no player block - https://gyazo.com/907725caaff0dd13f619cfe826b0e49d
2 stacked crates + no player block - https://gyazo.com/9f0ba9113e14d48765537bd4a15affe2
2 stacked crates + player block - https://gyazo.com/0bf55f563e20a3677e87f42b0db79d69
1 table + no player block - https://gyazo.com/d4a20fc76abc25a1ab1fa0387db36a89
1 table + player block - https://gyazo.com/f223e0ddb5a94536ef4ab28c760d7758
2 tables + no player block - https://gyazo.com/e2ed8b04556fa6793ad6e5a986fbbac2
2 tables + player block - https://gyazo.com/adf70d2aa04383c76d393de4f2b368dc

**Comment by nekiro** [comment]
Tfs has it correct, they only destroy the object when there is no free tile to push it to.

**Comment by waqmaz93** [comment]
if (moveCount < 20

**Comment by floki21uu** [comment]
> here is some old gifs i had (tests from experimental), mby you can get some information from them too
> 
> 1 crate + player block - https://gyazo.com/4b8ebad8b7d7c071c549983a250d7c25 1 crate + no player block - https://gyazo.com/907725caaff0dd13f619cfe826b0e49d 2 stacked crates + no player block - https://gyazo.com/9f0ba9113e14d48765537bd4a15affe2 2 stacked crates + player block - https://gyazo.com/0bf55f563e20a3677e87f42b0db79d69 1 table + no player block - https://gyazo.com/d4a20fc76abc25a1ab1fa0387db36a89 1 table + player block - https://gyazo.com/f223e0ddb5a94536ef4ab28c760d7758 2 tables + no player block - https://gyazo.com/e2ed8b04556fa6793ad6e5a986fbbac2 2 tables + player block - https://gyazo.com/adf70d2aa04383c76d393de4f2b368dc

Yo, did you fix it?:)
